### ISSUE TITLE
nginx file size limit seems low

### ISSUE THREAD

**Issue by insanity54** [sunsided/docker-nuget#3]
I was playing around with this and trying to push a .nupkg file that is 77MB. Got this error-

```
Failed to process request. 'Request Entity Too Large'.
The remote server returned an error: (413) Request Entity Too Large..
```

It looks like since client_max_body_size isn't set in https://github.com/sunsided/docker-nuget/blob/master/conf/nuget.conf, nginx sets it to a default of `20M` which is too low for my use case.

**EDIT**

I see the Dockerfile sets the file limit. What do you think about the filesize limit being settable via ENV variable, similar to how BASE_URL can be set?

**Comment by insanity54** [comment]
I was wrong, it isn't the Dockerfile that needs to set the size, because the Dockerfile is only used when the Docker image is created. Instead, a patch much like the ones in `scripts/` would be needed.

I went ahead and created this [feature](https://github.com/twowaynet/docker-nuget/blob/develop/scripts/patch-filesize.sh) in a [fork](https://github.com/twowaynet/docker-nuget), which adds environment variable `UPLOAD_MAX_FILESIZE`. If this is something you'd like upstream, I'd be happy to create a pull request.

**Comment by sunsided** [comment]
Absolutely, please do! :slightly_smiling_face:

**Comment by sunsided** [comment]
Merged #4.

**Comment by pauldotknopf** [comment]
This doesn't appear to be published on docker hub. Can you do a deploy for me?

Otherwise, I'll have to build a deploy my own image.
